### PR TITLE
Improve bookmark docs

### PR DESCRIPTION
**Pull request by FlorentinD**
Co-authored-by: Nicola Vitucci <nicola.vitucci@neo4j.com>

### CODE REVIEW

File: doc/modules/ROOT/pages/bookmarks.adoc [hunk 1]
```diff
@@ -3,13 +3,15 @@
 Bookmarks ensure that when executing queries in a Neo4j cluster, previous or parallel running queries are fully propagated through the cluster.
 By passing one or multiple bookmarks along with a query, the server will make sure that the query does not get executed before the represented state(s) have been established.
```
Comment (nvitucci) [inline doc/modules/ROOT/pages/bookmarks.adoc:4]:
```suggestion
A bookmark is a marker representing a state of the database.
Bookmarks are useful when subsequent transactions in a cluster need to be coordinated, for example when you need to read data right after writing it.

When using the GDS client, any query following a call to `gds.set_bookmarks()` is not executed until the bookmarked transactions are propagated across the cluster.
```

File: doc/modules/ROOT/pages/bookmarks.adoc [hunk 2]
```diff
@@ -3,13 +3,15 @@
...
 
-.Example of using bookmarks inside GDS
+.Example of using bookmarks with GDS
 [source, python]
 ----
 # If you ran queries through Neo4j driver before you can pass bookmarks in the constructor
```
Comment (nvitucci) [inline doc/modules/ROOT/pages/bookmarks.adoc:9]: I am not sure I understand the explanation here. Do you need to specify the `bookmarks=None` parameter here, or is `None` the default and you only need to specify it if you have created any bookmarks with the driver that you want to use?

**Review by nvitucci — COMMENTED** [review]
I've added some suggestions, hoping my understanding is correct.

File: doc/modules/ROOT/pages/bookmarks.adoc (hunk 1 above)
Comment (FlorentinD) [reply doc/modules/ROOT/pages/bookmarks.adoc:4]: Awesome ⭐

File: doc/modules/ROOT/pages/bookmarks.adoc (hunk 2 above)
Comment (FlorentinD) [reply doc/modules/ROOT/pages/bookmarks.adoc:9]: `None` is the default and yes, its if you created a driver instance before and now switch to the GDS client.

Comment (nvitucci) [reply doc/modules/ROOT/pages/bookmarks.adoc:11]: Ok, so what about rephrasing it as follows?

```suggestion
# Use the `bookmarks` parameter to set any existing bookmarks, for example
# after running a query through the Neo4j driver. The default value is `None`.
```

**Review by nvitucci — APPROVED** [review]

**Pull request merged**